src/services/ConfigApi.js

```diff
@@ -16,8 +16,8 @@ export class ConfigApi {
 		}
 	}
 
-	async validateSite(name, siteId) {
-		const apiPath = `${this.host}/api/customer/${siteId}/verify`;
+	async validateSite(siteId) {
+		const apiPath = `${this.host}/api/customer/${siteId}/verifyKey`;
 
 		const response = await fetch(apiPath, {
 			method: 'post',
```

Comment:
I believe it is no longer a `post` request.

https://github.com/searchspring/smc-config-api